src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.juddata.validators;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Set;
+
+public class AppointmentValidator implements ConstraintValidator<Appointment, String> {
+
+    private static final String VALUE1 = "CRTS TRIB - RS Admin User";
+    private static final String VALUE2 = "MAGS - AC Admin User";
+    private static final String VALUE3 = "Person on a List";
+    private static final String VALUE4 = "Unknown";
+
+    private static final Set<String> notAllowedValues = Set.of(VALUE1, VALUE2, VALUE3, VALUE4);
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        boolean valid = true;
+        if (StringUtils.isNotBlank(value)) {
+            valid = !notAllowedValues.contains(value.trim());
+        }
+        return valid;
```

Comment:
Very trivial .default return true  & may be remove boolean valid = true; remove 1 line code. if you think right update else keep as is.

Reply:
Could you please elaborate with a sample code ? Sorry, I didn't get that.

Reply:
Thanks for pointing that. Yes, this boolean variable is not needed. It has been changed.